src/Interpreter/Eval.hs

```diff
@@ -114,7 +116,8 @@ evalNumCompOp op expr1 expr2 scope = do
   (res2, _) <- evaluate expr2 scope
   case (res1, res2) of
     (Integer val1, Integer val2) -> Right $ (Bool $ op val1 val2, scope)
-    _ -> Left "Can compare only numbers"
+    (Char c1, Char c2) -> Right $ (Bool $ op (toInteger . ord $ c1) (toInteger . ord $ c2), scope)
+    _ -> Left "Can only compare two comparable types"
```

Comment:
Rename this function to `evalCompOp` as `Num` is unnecessary

Reply:
Done! Thanks for spotting this, I thought I had renamed it hehe